ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -64,7 +65,6 @@
   private static final String DELETE_TOPIC_ENABLE = "delete.topic.enable";
 
   private final AdminClient adminClient;
-  private final boolean isDeleteTopicEnabled;
```

Comment:
why are we no longer caching the result of this? I'm pretty sure that's a cluster-wide configuration

Reply:
It causes all commands to be denied if the user does not have `DescribeConfigs` permissions at the cluster level. No all users would need this operation, so I had to move it down to the moment the user attempts to delete the topic.